File: python/1 - Iniciante/1115_quadrante.py
while True:
    X, Y = map(int, input().split())

    if X == 0 or Y == 0:
        break

    if X != 0 and Y != 0:
        if X > 0 and Y > 0:
            print(f'primeiro')
        elif X < 0 and Y > 0:
            print(f'segundo')
        elif X < 0 and Y < 0:
            print(f'terceiro')
        else:
            print(f'quarto')

File: c/c99/1 - Iniciante/1115_quadrante.c
#include <stdio.h>

void quadrante(int A, int B) {
    if (A > 0 && B > 0) {
        printf("primeiro\n");
    } else if (A < 0 && B > 0) {
        printf("segundo\n");
    } else if (A < 0 && B < 0) {
        printf("terceiro\n");
    } else {
        printf("quarto\n");
    }
}

int main() {
    int X, Y;

    while (1) {
        scanf("%d %d", &X, &Y);

        if (X == 0 || Y == 0) {
            break;
        } else {
            quadrante(X, Y);
        }
    }

    return 0;
}
